pom.xml

```diff
@@ -227,19 +227,19 @@
     <dependency>
       <groupId>org.testcontainers</groupId>
       <artifactId>kafka</artifactId>
-      <version>1.15.3</version>
+      <version>1.19.0</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.testcontainers</groupId>
       <artifactId>testcontainers</artifactId>
-      <version>1.17.6</version>
+      <version>1.19.0</version>
```

Comment:
Please introduce a new variable for org.testcontainers group

Reply:
Updated